client/my-sites/domains/domain-management/list/all-domains.jsx

```diff
@@ -572,6 +573,12 @@ class AllDomains extends Component {
 				path: domainManagementRoot() + '?' + stringify( { filter: 'owned-by-others' } ),
 				count: filterDomainsByOwner( nonWpcomDomains, 'owned-by-others' )?.length,
 			},
+			{
+				label: translate( 'Parked domains' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( 'Parked Domain' )` *ES Score: 8*